index.js

```diff
@@ -1,4 +1,4 @@
-var WeakMap = global.WeakMap || require('core-js/library/fn/weak-map');
+var WeakMap = global.WeakMap || require('core-js/stable/weak-map');
```

Comment:
Honestly we can just remove this. I don't think we need to worry about IE10 anymore

Reply:
Hehe, I didn't realise that WeakMap was that old 😁 

Alternative PR: https://github.com/kolodny/weak-key/pull/6